hardhat/hardhat.config.js

```diff
@@ -19,7 +19,7 @@ module.exports = {
                 url: ALCHEMY_URL,
                 // Few blocks before WSTETH-A is taken at 14052147,
                 // https://etherscan.io/address/0x49a33a28c4c7d9576ab28898f4c9ac7e52ea457a
-                blockNumber: 14078340,
```

Comment:
I would not hardcode this change! Maybe document the new BlockNumber in this PR for later, but keep the original one. 

@LukSteib what do you think?

Reply:
Yes, fine for me